docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+Elasticsearch cluster, but Logstash can write to a large variety of {logstash}output-plugins.html[endpoints].
+
+// [float]
+[[deploying-minimal-install]]
+=== Getting Started: Minimal Installation
+
+The minimal Logstash installation has one Logstash instance and one Elasticsearch instance. These instances are 
+directly connected, with Logstash using an {logstash}input-plugins.html[_input plugin_] to ingest data and an 
+Elasticsearch {logstash}output-plugins.html[_output plugin_] to index the data in Elasticsearch, following the Logstash 
+{logstash}pipeline.html[_processing pipeline_]. A given Logstash instance has a fixed pipeline constructed at startup, 
+based on the instance’s configuration file. You must specify an input plugin. Output defaults to `stdout`, and the 
+filtering section of the pipeline, which is discussed in the next section, is optional.
+
+image::images/deploy_1.png[]
+
+// [float]
+[[deploying-filter-threads]]
+=== Adding a Filter and Threads
+
+Log data is typically unstructured, often contains extraneous information that isn’t relevant to your use case, and 
+sometimes is missing relevant information that can be derived from the log contents. A 
+{logstash}filter-plugins.html[filter plugin] can handle these situations by parsing the log into fields, removing 
+unnecessary fields, and adding new information derived from the existing fields. As an example, geolocation information 
+can be derived from an IP address and added to the logs, or arbitrary text can be parsed and structured by the 
+{logstash}plugins-filters-grok.html[grok] filter.
+
+Depending on the amount of computation the filter plugin requires, and the volume of the logs being processed, adding a 
+filter plugin can have a significant effect on performance. The grok filter’s regular expression computation is 
+particularly resource-intensive. One way to better address this increased demand for computing resources is to use 
+parallel processing on multicore machines. Use the -w switch to set the number of execution threads for Logstash 
+filtering tasks. For example the bin/logstash -w 8 command uses eight different threads for filter processing.
+
+image::images/deploy_2.png[]
+
+// [float]
+[[deploying-logstash-forwarder]]
+=== Using Logstash Forwarder
+
+The https://github.com/elastic/logstash-forwarder[Logstash Forwarder] is a lightweight, resource-friendly tool written 
+in Go that collects logs from files on the server for processing on other machines. The Logstash Forwarder acts as a 
+client to a centralized Logstash instance, communicating using a secure protocol called Lumberjack. The Logstash 
+instance that acts as a server must be configured to use the {logstash}plugins-inputs-lumberjack.html[Lumberjack 
+input plugin].
+
+The Logstash Forwarder uses the computing resources of the machine hosting the source data, and the Lumberjack input 
```

Comment:
I think the key points can be stated more clearly. Perhaps, "Logstash Forwarder provides a lightweight mechanism for collecting data from servers that are resource constrained. The Lumberjack input plugin is optimized so receiving the data doesn't have a big performance impact on your Logstash instance."